docs/Playbook-Variables.md

```diff
@@ -0,0 +1,38 @@
+| `gaiad_gov_testing`      | Set minimum deposit to `1` and voting period to `gaiad_voting_period` when `true` | `true`                                                                                              |
+| `gaiad_voting_period`    | Voting period for gov proposals                                                   | `60s`                                                                                               |
+| `enable_swap`            | Enable/disable swap                                                               | `false`                                                                                             |
+| `swap_size`              | Swap file size in MB (8 GB default)                                               | `8192`                                                                                              |
+| `cosmovisor_skip_backup` | Skip Cosmovisor backups                                                           | `true`                                                                                              |
+| `monitoring_prometheus`  | Configure Prometheus / Grafana monitoring                                         | `false`                                                                                             |
+| `gaiad_use_ssl_proxy`    | Enable SSL proxy using nginx to gaiad endpoints                                   | `false`                                                                                             |
+| `gaiad_api_host`         | Sets the subdomain for rest API (e.g. `rest.testnet.com`)**                       | `rest`                                                                                              |
+| `gaiad_rpc_host`         | Sets the subdomain for rpc (e.g. `rpc.testnet.com`)**                             | `rpc`                                                                                               |
+| `reboot`                 | If true, reboots the machine after all tasks are done***                          | `false`                                                                                             |
+| `monitoring_panic`       | Configure PANIC monitoring                                                        | `false`                                                                                             |
+| `panic_is_validator`     | Set host as a validator for PANIC                                                 | `no`                                                                                                |
+
+- *The file will not be copied if there already is an existing file with the same length.  
+- **Configure DNS before provisioning.  
+- *** Useful to make sure all services start up, recommended for initial deployment.
```

Comment:
The readme looks nice!

Should we also say `*** Useful to make sure all services start up and apply pending system updates, recommended for initial deployment.` since after the upgrades at the start of the playbook the system will also want a reboot.

Reply:
Thanks! I added the extra bit.